buildSrc/src/main/kotlin/Project.kt

```diff
@@ -8,7 +8,7 @@ object AndroidVersions {
     const val compileSdkVersion = 31
   }
   object Compose {
-    const val minSdkVersion = 23
+    const val minSdkVersion = 21
```

Comment:
Should this be 23?

Reply:
Thanks for the review @patrick-fitzgerald , according to Google's compose setup guide https://developer.android.com/jetpack/compose/setup, the minimum required SDK version is 21 for Jetpack Compose, is there any reason why should it be 23?

Reply:
Makes sense, it might be worth adding a comment. Thanks